common/src/main/java/com/microsoft/identity/common/internal/ui/webview/AzureActiveDirectoryWebViewClient.java

```diff
@@ -152,23 +155,59 @@ private boolean handleUrl(final WebView view, final String url) {
                 returnError(exception.getErrorCode(), exception.getMessage());
                 view.stopLoading();
             }
-            return true;
         } else if (isRedirectUrl(formattedURL)) {
             Logger.info(TAG, "Navigation starts with the redirect uri.");
-            return processRedirectUrl(view, url);
+            processRedirectUrl(view, url);
         } else if (isWebsiteRequestUrl(formattedURL)) {
             Logger.info(TAG, "It is an external website request");
-            return processWebsiteRequest(view, url);
+            processWebsiteRequest(view, url);
         } else if (isInstallRequestUrl(formattedURL)) {
             Logger.info(TAG, "It is an install request");
-            return processInstallRequest(view, url);
+            processInstallRequest(view, url);
         } else if (isWebCpUrl(formattedURL)) {
             Logger.info(TAG, "It is a request from WebCP");
-            return processWebCpRequest(view, url);
+            processWebCpRequest(view, url);
+        } else if (isPlayStoreUrl(formattedURL)){
+            Logger.info(TAG, "Request to open PlayStore.");
+            return processPlayStoreURL(view, url);
+        } else if (isAuthAppMFAUrl(formattedURL)){
+            Logger.info(TAG, "Request to link account with Authenticator.");
+            processAuthAppMFAUrl(url);
+        } else if (isInvalidRedirectUri(url)){
+            Logger.info(TAG, "Check for Redirect Uri.");
+            processInvalidRedirectUri(view, url);
+        } else if (isBlankPageRequest(formattedURL)){
+            Logger.info(TAG, "It is an blank page request");
```

Comment:
return false;?  //for blank page?

Reply:
so in this case, the meaning of the returning true, is we are servicing the urls and we have anticipated them. Whereas false, is when we come across urls which dont necessarily fit in the use cases we want to service.